workshops/collaborative_sketch/README.md

```diff
@@ -341,19 +421,28 @@ function clearDrawing() {
 }
 ```
 
-To make other screens clear when the drawing is removed from Firebase:
+Right now other screens won't clear when we remove the drawing from Firebase. To fix this, add the following:
 
 ```js
-pointsData.on("child_removed", function () {
-  points = [];
-});
+function setup() {
+  // ...the rest of the setup function
+  pointsData.on("child_added", function (point) {
+    points.push(point.val());
+  });
+  pointsData.on("child_removed", function () {
+    points = [];
+  });
+
+  canvas.mousePressed(drawPoint);
+  canvas.mouseMoved(drawPointIfMousePressed);
+}
 ```
 
 Congratulations! Your collaborative drawing app is now complete.
 
 ## Part IV: Publishing and Sharing
 
-Make sure your files are all saved, and then run the familiar git commands.
+Make sure your files are all saved, then run the familiar git commands.
 
 Then, remember to add `USERNAME.github.io` to the list of Authorized Domains on your Firebase app. After that, your project should be live on `USERNAME.github.io/collaborative_sketch/`!
```

Comment:
It would also address https://github.com/hackclub/hackclub/pull/895/files#r75043538